src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
-        expect(getFieldType(barField)).toBe(String);
-    });
+            expect(getFieldType(fooField)).toBe(String);
+        });
+        it('should get field type from field when type is array of type', () => {
+            const fooField = {
+                type: [String],
+            };
 
-    it('should return function name', () => {
-        const barFunction = function bar() {};
-        const fooFunction = function foo() {};
-        Object.defineProperty(fooFunction, 'name', { value: undefined });
-        expect(getFunctionName(barFunction)).toBe('bar');
-        expect(getFunctionName(String)).toBe('String');
-        expect(getFunctionName(Number)).toBe('Number');
-        expect(getFunctionName(Object)).toBe('Object');
-        expect(getFunctionName(Array)).toBe('Array');
-        expect(getFunctionName(Boolean)).toBe('Boolean');
-        expect(getFunctionName(Date)).toBe('Date');
-        expect(getFunctionName(fooFunction)).toBe('foo');
+            expect(getFieldType(fooField)).toBe(String);
+        });
+    });
+    describe('getFunctionName', () => {
+        it('should return function name of String', () => {
+            expect(getFunctionName(String)).toBe('String');
+        });
+        it('should return function name of Number', () => {
+            expect(getFunctionName(Number)).toBe('Number');
+        });
+        it('should return function name of Object', () => {
+            expect(getFunctionName(Object)).toBe('Object');
+        });
+        it('should return function name of Array', () => {
+            expect(getFunctionName(Array)).toBe('Array');
+        });
+        it('should return function name of Boolean', () => {
+            expect(getFunctionName(Boolean)).toBe('Boolean');
+        });
+        it('should return function name of Date', () => {
+            expect(getFunctionName(Date)).toBe('Date');
+        });
+        it('should return function name of fooFunction', () => {
+            const fooFunction = function foo() {};
+            Object.defineProperty(fooFunction, 'name', { value: undefined });
+            expect(getFunctionName(fooFunction)).toBe('foo');
+        });
+        it('should return function name of barFunction', () => {
+            const barFunction = function bar() {};
+            expect(getFunctionName(barFunction)).toBe('bar');
+        });
+    });
+    describe('removeFirstKeyIfNumber', () => {
+        it('should remove first key if is number', () => {
+            const keys = ['0', 'foo', 'bar'];
+            expect(removeFirstKeyIfNumber(keys)).toEqual(['foo', 'bar']);
+        });
+        it('should not remove first key if is string', () => {
+            const keys = ['first', 'foo', 'bar'];
+            expect(removeFirstKeyIfNumber(keys)).toEqual(['first', 'foo', 'bar']);
+        });
+    });
+    describe('getErrorIndexFromKeys', () => {
+        it('should return first key as number if is number', () => {
```

Comment:
`should return first key as number if is number` --> `should return first key as a number if it is a number`

Reply:
changed